btc-lsp/src/BtcLsp/Storage/Model/Block.hs

```diff
@@ -67,3 +68,11 @@ getLatestSql =
       [ Psql.Desc BlockHeight,
         Psql.LimitTo 1
       ]
+
+getBlock :: (MonadIO m) => BlkHeight -> ReaderT Psql.SqlBackend m [Entity Block]
+getBlock blkHeight = do
```

Comment:
Just small style issue - in every other place we are using `*Sql` naming for requests, probably should be the same `getBlockByHeightSql`.